.papr.yml

```diff
@@ -8,9 +8,11 @@ cluster:
 context: fedora/26/atomic
 
 packages:
-  - ansible
+  - python-virtualenv
```

Comment:
Let's add `libselinux-python` to this list, which is used when Ansible writes out files to the host.  We probably don't need this for PAPR, but it wouldn't hurt.

Reply:
I added `libselinux-pytho`n and also the ` --system-site-packages` option.  Looks to be working from the checks passing!